addons/binding/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/api/IBaseFroniusResponse.java

```diff
@@ -0,0 +1,11 @@
+package org.openhab.binding.fronius.internal.api;
+
+/**
+ * Basic interface for a response-object from the API
+ * 
+ * @author rokohl
+ *
+ */
+public interface IBaseFroniusResponse {
```

Comment:
I think you can implement this as a class an implement the `getHead()` here, because it's the same for both classes extending this interface. You might check if that works correctly with gson parsing.